cumulus-client/build.gradle

```diff
@@ -2,7 +2,13 @@ dependencies {
     implementation(project(":cumulus-model"))
     implementation("mil.army.usace.hec:cwms-http-client:1.0-SNAPSHOT")
 
+    testImplementation("mil.army.usace.hec:cwms-http-client:1.0-SNAPSHOT")
     testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.1")
+    testImplementation('mil.army.usace.hec:cwms-http-client:1.0-SNAPSHOT:test-fixtures@jar')
+    testImplementation(platform("com.squareup.okhttp3:okhttp-bom:4.9.2"))
```

Comment:
why do you have to depend on okhttp? cwms-http-client shouldnt be exposing okhttp deps

Reply:
The TestController class needs the testImplementation('com.squareup.okhttp3:mockwebserver:4.9.2'). But I can remove the other dependencies.

Reply:
I dont see any imports from okhttp in TestController

Reply:
I didn't think I needed it either, but if I don't I get this: 

okhttp3/mockwebserver/MockWebServer
java.lang.NoClassDefFoundError: okhttp3/mockwebserver/MockWebServer
	at mil.army.usace.hec.cwms.htp.client.MockHttpServer.create(MockHttpServer.java:40)
	at mil.army.usace.hec.cumulus.client.controllers.TestController.setUp(TestController.java:43)

Caused by: java.lang.ClassNotFoundException: okhttp3.mockwebserver.MockWebServer

Reply:
Oh, that means its runtime dependent but not compile time dependent on test. I wonder if there is a way to set that in gradle?

Reply:
Oh yeah, I changed it to testRuntimeOnly('com.squareup.okhttp3:mockwebserver:4.9.2') and I don't get the error

Reply:
Not sure. It definitely can be set to a testRuntimeOnly dependency since its only needed for the TestController, but i'm not sure what would need to be done to remove the dependency completely here and have it still work

Reply:
Seems to me that the pom from cwms-http-client doesn't include everything that it needs to use its test framework, hence we are looking behind the curtain and including an cwms-http-client internal dependency here. I could be wrong though.

Reply:
Seems to be a gradle "feature" to not provide testFixtureImplementation dependencies:
https://github.com/gradle/gradle/issues/16011

Reply:
Updated to a testRuntimeOnly dependency